src/main/java/flaxbeard/immersivepetroleum/client/gui/elements/PipeConfig.java

```diff
@@ -116,47 +114,43 @@ public void updateTexture(){
 		for(int gy = 0;gy < this.grid.getHeight();gy++){
 			for(int gx = 0;gx < this.grid.getWidth();gx++){
 				int color = 0;
-				
-				switch(this.grid.get(gx, gy)){
-					case EMPTY:{
-						if((gx >= texCenterX - 2 && gx <= texCenterX + 2) && (gy >= texCenterY - 2 && gy <= texCenterY + 2)){
+
+				switch (this.grid.get(gx, gy)) {
```

Comment:
As with `(){` having no space, keywords like `switch()` don't have spaces between them either.

Same with `if`, `else`, `try`, `catch` and all the others.